### DISCUSSION TITLE
Error reporting could be more specific (issues with Synology remote repo)

### DISCUSSION THREAD

**Discussion by palijn** [borgbase/vorta#1264] [Ideas]
Hi,
I've battled quite a bit setting up a remote repository on my Synology NAS. 
Borg was properly installed through a SynoCommunity package, Vorta seemed to work fine, but it kept telling me either "Connection closed" or "Error…Is borg running on the server?", depending on the user@nas.local:/path/to/repo combination.
Vorta logs saved me because it told me what command line was used : that is a great 👍 for Vorta !

As a suggestion, Vorta could have been more helpful in the GUI, in that :
-the connection closed issue was when I used a remote user who was not in the administrators group, hence not ssh-able. About this, Vorta cannot do anything I think, although a note in the installation documentation would be nice ?
-the "is borg running?" question **could** have been much clearer. Especially since it did not require anything "running" on the server side.
Indeed, by running myself the ssh borg.exe command adding the --debug flag, it clearly appeared that remote sh did actually output an error message : the borg command was not found !
For the record, that's a PATH issue that is not the same on interactive than on non-interactive sessions - one that is easily fixed with a SetEnv directive in the sshd configuration.

I'm just reporting this here because : 
- it might be useful to update the doc/the logging
- Synology users could find this discussion and solve their issue ;)

**Comment by real-yfprojects** [comment]
I think, the error message originates directly from borg. This might therefore be their issue. Though logs might provide a better understanding of the issue.

**Reply by palijn** [reply]
Well, the error message is clear enough to me. I don't even think that borg could have access to more information since that's about all that ssh will tell.
It is the fact that his useful message is buried deep into the Vorta logs that is more of an issue IMHO : had it surfaced, it would have been a trivial fix.

**Reply by real-yfprojects** [reply]
If you enclose the logs. I can understand in which order and severity borg logs the error messages.

**Reply by palijn** [reply]
I am not sure I understand what you need. To replicate the issue, simply move the borg executable on the server side outside the PATH, run a borg command from Vorta, and look into the Vorta log ?

**Reply by real-yfprojects** [reply]
I don't have a borg server available and no time to set one up. Enclosing the logs when opening an issue on the other hand is quite simple.
